app/controllers/application_controller.rb

```diff
@@ -36,4 +37,10 @@ def layout_by_resource
   def available_roles
     current_user.admin? ? User::ROLES.reject { |el| el.eql?(User::SUPER_ADMIN_ROLE) } : User::ROLES
   end
+
+  private
+
+  def set_timezone
+    Time.zone = ActiveSupport::TimeZone['Kyiv']
```

Comment:
in applicaiton.rb time zone is specified? 
if yes - remove it from here.
if not - add and then remove from here.